.travis.yml

```diff
@@ -23,7 +23,7 @@ before_install:
   - cd ../..
 
 script:
-  - "mvn verify -Dei.backendInstancesListJsonContent=\"[{ 'contextPath': '', 'port': '${EI_BACKEND_PORT}', 'name': 'EI-Backend-1', 'host': 'localhost', 'https': false, 'defaultBackend': true}]\" -B"
```

Comment:
I think we should change the ei.backendInstancesListJsonContent property to only lower letter or only capital letter and split the words with dot character: ei.backend.instances.list.json.content

Reply:
It is going to be in a separate commit